<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>control</class>
 <widget class="QWidget" name="control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">control
{
	background-color: rgb(0, 0, 0);
}
QPlainTextEdit
{
	color: rgb(0, 0, 0);
	background-color: rgb(223, 225, 106);
}
QPushButton
{
	color: rgb(237, 212, 0);
	background-color: rgb(0, 0, 0);
	border-radius:16px;
	border:2px solid yellow;
}
QPushButton:hover
{
	color: rgb(238, 238, 236);
	background-color: rgb(237, 212, 0);
}
	</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>631</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="show_sans">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="show_names">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="show_movies">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPlainTextEdit" name="show_reserved">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_names">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Names</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_movies">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Movies</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_sans">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Sans</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_reserved">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Reserved</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
